### DISCUSSION TITLE
how to get max response time after test?

### DISCUSSION THREAD

**Discussion by wenhuiwu** [naver/ngrinder#879] [Q&A]
how can i get the max response time after test? or 90% response time of the test?

**Comment by donggyu04** [comment]
@wenhuiwu 

Hi, It is not provided by default.
you can use [user defined statistic](https://github.com/naver/ngrinder/wiki/Multiple-User-Defined-Statistic-In-nGrinder) to record the response time(measure time before and after API call) and view it in a report page or download it as a CSV.
